overrides/resources/crafttweaker/lang/en_us.lang

```diff
@@ -666,17 +665,17 @@ dj2.bio_velocity.desc0=§f- Solifuges (in the Ulterior Outback biome)
 dj2.bio_velocity.desc1=§f- Centipedes (in most biomes)
 dj2.supernatural_velocity.desc0=§f- Solifuges (in the Ulterior Outback biome)
 dj2.supernatural_velocity.desc1=§f- Centipedes (in most biomes)
-dj2.exoskeleton_plate.desc0=§f- Beetles (in Underground Jungles,
-dj2.exoskeleton_plate.desc1=§f           Subterranean Savannahs and Elysian Fields)
-dj2.exoskeleton_plate.desc2=§f- Antlions (in Volcanic Deserts and in Antlion Overlord Pyramids)
+dj2.exoskeleton_plate.desc0=§f- Beetles (in the Underground Jungle,
+dj2.exoskeleton_plate.desc1=§f           Subterranean Savannah, and Elysian Fields biomes)
+dj2.exoskeleton_plate.desc2=§f- Antlions (in Volcanic Desert biome and in Antlion Overlord Pyramids)
```

Comment:
uh, Antlion Overlord Pyramids exclusively spawn in the Volcanic Desert biome. should delete that part